### DISCUSSION TITLE
App store?

### DISCUSSION THREAD

**Discussion by Djspac3** [geode-sdk/sdk#639] [Help] [answered]
Just is geode going to be on an app store like Google play store and window's whatever?

**Comment by Netherbat** [comment]
no because apple

**Reply by Djspac3** [reply]
... I NEVER ASKED ABOUT APPLES APP STORE I ASKED ABOUT GOOGLE OR WINDOWS WHATEVER

I ALREAD LOOKED AT APPLE AND THAT YOU CAN'T EVEN ADD IT

**Reply by Alphalaneous** [reply, marked as answer]
Geode will not be on any platform app stores as it cannot abide to the guidelines due to the nature of it.

**Reply by Djspac3** [reply]
Dang it

**Reply by Djspac3** [reply]
Tho how can skeld.net be on the app store for android (Google play store)
